src/app/schemas/loginSchema.ts

```diff
@@ -0,0 +1,11 @@
+// src/schemas/login.schema.ts
+import { z } from "zod";
+
+export const loginSchema = z.object({
+  email: z
+    .string()
```

Comment:
```suggestion
    .string({
    message:"dfdefgrergb"
    })
```
文字列空のメッセージってこうしないとでてこなかったような

Reply:
[177e786](https://github.com/wato787/codex-frontend/pull/7/commits/177e786250df289e4b7966e217d9e141a1115c95)